catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -124,11 +124,9 @@
 
   public static final int GET_BY_ID_LIMIT = 100;
 
-  public static final String EXCLUDE_ATTRIBUTES = "excludeAttributes";
```

Comment:
Why was this variable removed? Since it is a `public` static it would be worth checking if this is used by any downstream projects before removing.

Reply:
I looked at when it was added and when it was removed, looked like they forgot to remove it. I did a search in ddf project and all other projects in codice, and none were referring to it.

EXCLUDE_ATTRIBUTES was added in this PR
https://github.com/codice/ddf/pull/2355/files#diff-68cb34fde870638e3f967e7ea860fe2410ef6881b98aab7d87bbe1bab851aec3R94

and was removed in this PR
https://github.com/codice/ddf/pull/6331/files#diff-2f7227781cbf22a26262018529f7747aae8460a7f98ca6da4c983f46bea27f1cL686

Also note they converted SolrProviderTest.java into a Suite and moved its content to SolrProviderQuery.java (EXCLUDE_ATTRIBUTES was referred here)

Reply:
Cool thanks @abel-connexta that soudns fine. @jlcsmith, @pklinef do you know if this EXCLUDE_ATTRIBUTES variable was intentionally left here? If not I'm okay with removing it.

Reply:
At this point I'm thinking we can remove. I searched some downstream projects and couldn't find this exact variable being referenced.

Reply:
Agree to remove this